packages/web/src/components/notification/Notifications/UserNameLink.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { MouseEventHandler, useCallback } from 'react'
+
+import cn from 'classnames'
+import { push } from 'connected-react-router'
```

Comment:
super-duper nit but I think we typically import this `as pushRoute` for clarity?